hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataReader.java

```diff
@@ -91,50 +88,19 @@ public OmMetadataReader(KeyManager keyManager,
                           Logger log,
                           AuditLogger audit,
                           OmMetadataReaderMetrics omMetadataReaderMetrics,
-                          boolean bNeedAccessAuthInitialization) {
+                          IAccessAuthorizer accessAuthorizer) {
     this.keyManager = keyManager;
     this.bucketManager = ozoneManager.getBucketManager();
     this.volumeManager = ozoneManager.getVolumeManager();
     this.prefixManager = prefixManager;
-    OzoneConfiguration configuration = ozoneManager.getConfiguration();
     this.ozoneManager = ozoneManager;
     this.isAclEnabled = ozoneManager.getAclsEnabled();
     this.log = log;
     this.audit = audit;
-    boolean allowListAllVolumes = ozoneManager.getAllowListAllVolumes();
     this.metrics = omMetadataReaderMetrics;
     this.perfMetrics = ozoneManager.getPerfMetrics();
-    if (isAclEnabled) {
-      Class<? extends IAccessAuthorizer> clazz = configuration.getClass(
-          OZONE_ACL_AUTHORIZER_CLASS, OzoneAccessAuthorizer.class,
-          IAccessAuthorizer.class);
-      if (bNeedAccessAuthInitialization ||
-          clazz.getSimpleName().equals(
-              OzoneNativeAuthorizer.class.getSimpleName())) {
-        // In case of NativeAuthorizer always re-initialize
-        accessAuthorizer = getACLAuthorizerInstance(configuration, clazz);
```

Comment:
We can delete unused method `getACLAuthorizerInstance` as well.

Reply:
Good point, thanks.